app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,20 @@
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    user = User.find_by email: params[:session][:email].downcase
+
+    if user&.authenticate(params[:session][:password])
```

Comment:
Cú pháp `&.` để làm j vậy

Reply:
viết gộp lại từ if user && user.authenticate(params[:session][:password])

Reply:
Chị bjk. Cơ mà vú pháp này ý nghĩ là j.
Search toán tử an toàn với rails để đọc thêm

Reply:
Ok chị. Em cảm ơn.